### PR TITLE
Add incremental CSV upload

### PR DESCRIPTION
**Pull request by alexgleith**
- Adds a simple query to COPY command to do an incremental CSV export
- Tested only a little bit

### CODE REVIEW

**Comment by whatnick** [comment]
Can we try these incremental CSV's in a separate DAG in develop first ?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,49 +1,90 @@
...
+      test: ["CMD-SHELL", "[ -f /opt/airflow/airflow-scheduler.pid ]"]
       interval: 30s
       timeout: 30s
       retries: 3
   maildev:
     image: maildev/maildev
     ports:
       - "1080:80"
+volumes:
+  pgdata:
```
Comment (alexgleith) [inline docker-compose.yml:89]: I wouldn't worry about a volume here.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,49 +1,90 @@
...
+  logs:
```
Comment (alexgleith) [inline docker-compose.yml:90]: Or here. Having logs go to a directory might be useful, but that can be done with a volume mount and doesn't need to be specified.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,49 +1,90 @@
...
+      options:
+        max-size: 10m
+        max-file: "3"
+    volumes:
+      - ./dags:/opt/airflow/dags
+      - ./plugins:/opt/airflow/plugins
+      - ./scripts:/opt/airflow/scripts
+      - logs/:/opt/airflow/logs
+      - ./requirements.txt:/opt/airflow/requirements.txt
+      - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
```
Comment (alexgleith) [inline docker-compose.yml:75]: Docker's random isn't random?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,49 +1,90 @@
...
+    environment: *env
+    logging:
+      options:
+        max-size: 10m
+        max-file: "3"
+    volumes:
+      - ./dags:/opt/airflow/dags
+      - ./plugins:/opt/airflow/plugins
+      - ./scripts:/opt/airflow/scripts
+      - logs/:/opt/airflow/logs
```
Comment (alexgleith) [inline docker-compose.yml:73]:
```suggestion
      - ./logs/:/opt/airflow/logs
```
And make the logs dir with a .gitkeep would work, right?

File: docker-compose.yml [hunk 5]
```diff
@@ -1,49 +1,90 @@
...
-    image: postgres:9.6
+    image: postgres:12
     environment:
       - POSTGRES_USER=airflow
       - POSTGRES_PASSWORD=airflow
       - POSTGRES_DB=airflow
       - PGDATA=/var/lib/postgresql/data/pgdata
     volumes:
-      - ./pgdata:/var/lib/postgresql/data/pgdata
+      - pgdata:/var/lib/postgresql/data/pgdata
```
Comment (alexgleith) [inline docker-compose.yml:11]: For dev/test, I'd just let this slide away! It sticks around without a volume until you delete the container.

**Comment by omad** [comment]
I'm going to close this, and replace it with #62. The CSV upload is in a
separate DAG, has been tested locally, and includes all tables.

**Pull request closed without merging**